constitution.tex

```diff
@@ -390,6 +391,16 @@
 	\item To ensure that there is a variety of activities for House members to participate in throughout the academic year
 \end{enumerate}
 
+\asubsection{Responsibilities of the Public Relations Director}
+\begin{enumerate}
+	\item To preside over Public Relations Directorship Meetings
+	\item To operate House social media accounts intended to represent House as a whole
+	\item To oversee the organization, initiation, and execution of House philanthropic events
+	\item To preserve and improve the public image of House
+	\item To collaborate with other Executive Board members to organize, initiate, execute, and advertise any events that are intended to be attended by RIT Students whom are not Computer Science House Members and/or are Introductory Members
```

Comment:
Nit: introductory members are included in "Members"

Also "Computer Science House" here can be ellided

Just generally this section needs a reword imo... The language is awkward.